Riya Singhal/Golang/generalSyntax.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import "fmt"
+
+func main() {
+
+	fmt.Println("Hello world")
+
+	// explicit type
+	var number uint16 = 31
+	fmt.Println("Number is:", number)
+
+	// implicit type
+	var number2 = 31
+	fmt.Println("Number is:", number2)
+
+	//%T determines datatype
```

Comment:
```suggestion
	//%T determines datatype
```
don't start commenting with something like this `%T`